src/expressions/loop-expr.md

```diff
@@ -36,12 +36,10 @@ Only `loop` supports [evaluation to non-trivial values](#break-and-loop-values).
 A `loop` expression repeats execution of its body continuously:
 `loop { println!("I live."); }`.
 
-A `loop` expression without an associated `break` expression is
-[diverging](items/functions.html#diverging-functions), and doesn't
-return anything. A `loop` expression containing associated
-[`break` expression(s)](#break-expressions)
-may terminate, and must have type compatible with the value of the `break`
-expression(s).
+A `loop` expression without an associated `break` expression is diverging and
+so has type `!`. A `loop` expression containing associated 
```

Comment:
Can you link to the new `Never type` section you created?

And can you rephrase the end of the first sentence to one of these?
- "expression is divergent and has type `!`."
- "expression diverges and has type `!`."

"and so" is redundant.